sunpy/net/dataretriever/tests/test_noaa.py

```diff
@@ -8,9 +10,51 @@
 from sunpy.net.fido_factory import UnifiedResponse
 from sunpy.net import Fido
 from sunpy.net import attrs as a
+
 LCClient = noaa.NOAAIndicesClient()
 
 
+def create_mock_unified_object(start_date, end_date):
```

Comment:
Could we rename this to something a bit shorter?

Reply:
`mock_query_response()` ?

Reply:
sure